app/src/main/java/org/oppia/android/app/shim/ViewBindingShimImpl.kt

```diff
@@ -63,6 +65,27 @@ class ViewBindingShimImpl @Inject constructor() : ViewBindingShim {
     return DataBindingUtil.findBinding<ProfileInputViewBinding>(profileInputView)!!.errorText
   }
 
+  override fun inflatePromotedStoryCardBinding(
+    inflater: LayoutInflater,
+    parent: ViewGroup,
+    attachToParent: Boolean
+  ): View {
+    return PromotedStoryCardBinding.inflate(
+      LayoutInflater.from(parent.context),
+      parent,
+      /* attachToParent= */ attachToParent
```

Comment:
Comment isn't needed anymore since the variable describes the context.

Reply:
Done